artemis-server/src/main/java/org/apache/activemq/artemis/core/settings/impl/AddressSettings.java

```diff
@@ -1479,7 +1500,8 @@ public int getEncodeSize() {
          BufferHelper.sizeOfNullableBoolean(defaultGroupRebalancePauseDispatch) +
          BufferHelper.sizeOfNullableInteger(managementMessageAttributeSizeLimit) +
          BufferHelper.sizeOfNullableInteger(slowConsumerThresholdMeasurementUnit.getValue()) +
-         BufferHelper.sizeOfNullableBoolean(enableIngressTimestamp);
+         BufferHelper.sizeOfNullableBoolean(enableIngressTimestamp) +
```

Comment:
Typo here. Should be something like `BufferHelper.sizeOfNullableLong(maxMessages)`.